### PR TITLE
Restyle FLE home page, text styles, inner nav bar styles, button styl…

### PR DESCRIPTION
**Pull request by khangmach**
Restyle FLE home page, text styles, inner nav bar styles, button styles, tighten container heights, new images

### CODE REVIEW

**Comment by jtamiace** [comment]
screen shot 2017-04-04 at 9 47 49 am


I wonder if we should choose a different photo or edit the existing one... the text is not contrasted well

**Comment by jtamiace** [comment]
Liking the new color scheme for buttons / inner nav!

**Comment by khangmach** [comment]
@jtamiace gonna make edits later to the image to make it a bit darker so that there is some sorta contrast

File: fle_site/apps/main/static/css/homepage.css [hunk 1]
```diff
@@ -53,29 +52,30 @@ a:hover {
...
     width: 100%;
-    box-shadow: 1px 0px 10px 5px #000000;
+    box-shadow: 1px 0px 5px 2px #898989;
     visibility: visible;
 }
 .active a,
 #fle-inner-nav li:hover {
-	background: #557284 !important;
-	box-shadow: inset 0 0 8px #1f1f1f;
+	background: #00AEF0 !important;
```
Comment (jtamiace) [inline fle_site/apps/main/static/css/homepage.css]: the frequency of use of !important makes me cringe... (just reprimanding past me)

File: fle_site/apps/main/static/css/homepage.css [hunk 2]
```diff
@@ -87,24 +87,42 @@ a:hover {
...
+	background-color:#3a7aa2;
+}
+
+
+/*=========== BOOTSTRAP NAV STYLE OVERWRITE===============*/
+
+.navbar-default .navbar-toggle{
+	border:none;
+}
+/*
```
Comment (jtamiace) [inline fle_site/apps/main/static/css/homepage.css]: feel free to remove dead code

Comment (jtamiace) [inline fle_site/apps/main/static/css/homepage.css]: (commented out code)

File: fle_site/apps/main/static/css/homepage.css [hunk 3]
```diff
@@ -629,21 +648,21 @@ a:hover {
 }
 @media (max-width: 991px) {
 	#landing-header {
-		font-size: 20px;
+		font-size: 28px; /*prev 20*/
```
Comment (jtamiace) [inline fle_site/apps/main/static/css/homepage.css]: can remove comment

**Comment by jtamiace** [comment]
(Just posting a thought for the future)

The blog post links in that carousel are quite outdated... would be good at a later time to see if there's some sort of integration with medium to get latest/popular posts showing up on the home page

**Comment by jtamiace** [comment]
screen shot 2017-04-04 at 10 04 52 am

(reprimanding of past self intensifies)

**Comment by khangmach** [comment]
oh LOL the icon and links two separate rows

**Comment by khangmach** [comment]
@jtamiace updated the PR :-)

**Comment by jtamiace** [comment]
That footer is fresh

**Comment by khangmach** [comment]
footer success subscription interaction took so long to implement

**Comment by jtamiace** [comment]
There's a little bit of an HTML comment on the top left of this photo :O might be the cause of breaking the layout of the blog/press squares a bit?

screen shot 2017-04-07 at 10 26 42 am

**Comment by khangmach** [comment]
Fixed lol

**Comment by jtamiace** [comment]
Awesome. Changes look great. Merging!!!! :D

**Pull request merged**